scripts/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Running command example 
+# docker run tox-node 1D5A5F2F5D6233058BF0259B09622FB40B482E4FA0931EB8FD3AB8E7BF7DAF6F 198.98.51.198:33445 bf042bb227d0c7ff38302ead95a9e97e719182898e1cab8eab2ab6cda81de367
+#
+# arguments are bootstrap's publickey, Ip:port, own secretkey
+
+FROM rust:latest
```

Comment:
Use multi-stage build to separate build and running: https://docs.docker.com/v17.09/engine/userguide/eng-image/multistage-build/